doc/integrator/install_application.rst

```diff
@@ -165,6 +167,23 @@ using a C compiler.
 Furthermore, some changes in the apache wsgi and mapserver configurations are
 required to make c2cgeoportal work on Windows.
 
+Also, between all the different command interfaces available on windows (cmd, 
+Cygwin, git mingw), only Windows default cmd interface handle paths correctly
+in all stage of the application setup.
+
+Command interface and environment variable
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Only use Windows default commande interface::
+
+    Start > Run... > cmd
+
+Cygwin and git mingw are not compatible. Powershell is untested.
+
+Complementarily you need to add all the ressource paths to your system PATH
+environment variable, for cygwin, git, node, apache, postgres, mapserver and
```

Comment:
On 22.01.2016 07:52, Michael Kalbermatten wrote:
> In doc/integrator/install_application.rst
> <https://github.com/camptocamp/c2cgeoportal/pull/1945#discussion_r50506615>:
> 
>> @@ -165,6 +167,23 @@ using a C compiler.
>>  Furthermore, some changes in the apache wsgi and mapserver configurations are
>>  required to make c2cgeoportal work on Windows.
>>  
>> +Also, between all the different command interfaces available on windows (cmd, 
>> +Cygwin, git mingw), only Windows default cmd interface handle paths correctly
>> +in all stage of the application setup.
>> +
>> +Command interface and environment variable
>> +^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>> +
>> +Only use Windows default commande interface::
>> +
>> +    Start > Run... > cmd
>> +
>> +Cygwin and git mingw are not compatible. Powershell is untested.
>> +
>> +Complementarily you need to add all the ressource paths to your system PATH
>> +environment variable, for cygwin, git, node, apache, postgres, mapserver and
> 
> ping @ochriste <https://github.com/ochriste>
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/camptocamp/c2cgeoportal/pull/1945/files#r50506615>.
> 

I updated the PR, apache, postgres, mapserver were removed, only cygwin,
git, node are left.


-- 
Camptocamp SA
Oliver Christen
PSE A, EPFL
CH-1015 Lausanne

www.camptocamp.com

+41 21 619 10 23 (direct)
+41 21 619 10 10 (centrale)
+41 21 619 10 00 (fax)

---
L'absence de virus dans ce courrier électronique a été vérifiée par le logiciel antivirus Avast.
https://www.avast.com/antivirus